crates/formatter/tests/cases/issue_130/before.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+function getControls(): array
+{
+    return [
+        ...($this->bufferEnabled ? ['esc' => 'select'] : ['/' => 'filter', 'space' => 'select']),
```

Comment:
does not resolve #130, we just noticed that parens were missing in this case, and fixes it here.